hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/metrics/management/ReadMetricsOperation.java

```diff
@@ -16,22 +16,20 @@
 
 package com.hazelcast.jet.impl.metrics.management;
 
-import com.hazelcast.core.OperationTimeoutException;
+import com.hazelcast.jet.impl.JetService;
 import com.hazelcast.jet.impl.metrics.JetMetricsService;
 import com.hazelcast.jet.impl.metrics.management.ConcurrentArrayRingbuffer.RingbufferSlice;
-import com.hazelcast.spi.BlockingOperation;
 import com.hazelcast.spi.Operation;
 import com.hazelcast.spi.ReadonlyOperation;
-import com.hazelcast.spi.WaitNotifyKey;
 
 import java.util.Map.Entry;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.TimeUnit;
 
-public class ReadMetricsOperation extends Operation implements BlockingOperation, ReadonlyOperation {
+public class ReadMetricsOperation extends Operation implements ReadonlyOperation {
 
     private static final int MIN_TIMEOUT_SECONDS = 5;
```

Comment:
I think we need a decision to keep timeout on the operation or not ?

Reply:
yes timeout should be removed